source/Projectile.h

```diff
@@ -105,6 +105,9 @@ class Projectile : public Body {
 	const Ship *cachedTarget = nullptr;
 	const Government *targetGovernment = nullptr;
 
+	// How much velocity this projectile added to itself on top of the
+	// velocity of its parent.
+	double marginalVelocity = 0.;
```

Comment:
Thanks for adding this; I didn't want to/know exactly how to play with the class definition.  That being said, I don't know if a `marginalVelocity` for each step is as helpful as a `totalProjectileVelocity` or `totalProjectileSpeed` attribute would be.

Reply:
Well the total is just the velocity.Length(), is it not?